.github/workflows/windows.yml

```diff
@@ -120,17 +137,49 @@ jobs:
           OPENENROTH_MM7_PATH: 'D:\a\OpenEnroth\OpenEnroth\OpenEnroth_GameData\mm7'
 
       - name: Prepare artifact
+        if: inputs.releaseTag != ''
         shell: pwsh
-        run: |
+        run: >
           mkdir dist
+
           copy build\src\Bin\OpenEnroth\OpenEnroth.exe dist\ 
-          xcopy resources dist\ /E/H/C/I
 
-#      - uses: actions/upload-artifact@v3
-#        with:
-#          name: ccache-debug
-#          path: C:\debug
+          if ("${{ matrix.configuration }}" -eq "Debug" -And "${{
```

Comment:
this all isnt needed anymore with static binaries

Reply:
Yeah, but we haven't merged those yet, I want to have a look at why static release builds aren't working